common/diplo_phrases/hfe_diplo_phrases.txt

```diff
@@ -213,8 +220,10 @@ greetings = {
 		trigger = {
 			has_country_flag = hfe_egypt
 			from = {
-				any_relation = {
-					has_country_flag = hfe_future
+				any_owned_pop = {
```

Comment:
No, its not how it intended

Reply:
The localization says "humans that aren't us" and has said so since I joined this Mod's development board. When YOU, not the Kad Republic, talks to the Ju Ahet, YOU are originally told "oh look, Kad humans, how primitive", _despite having no Kad Humans among you_. In the English language, the second person "you", in its singular form, refers to what YOU have. This is a translation issue. Maybe Russian works differently?

Reply:
Original line is
Kad? How primitive.

Reply:
The Ju Ahets would say that _to the Kad Republic_, not to the player.

Reply:
No

Reply:
The Ju Ahets would say that to the KAD REPUBLIC, not the player. In the English language, the second person "you", in its singular form, refers to what YOU have. You, the player, are not the Kad Republic.

This is a translation issue and a problem with language. I'm not changing it back because the original line was stupid.

Reply:
Then text should be corrected

Reply:
I already did.

Reply:
Unresolved

Reply:
I already fixed the text. Also, I made the changes to the Localization MONTHS AGO, way before I EVER made any suggestion. If you keep whining about this, then we should just delete this phrase or make a new one THAT DOESN'T SUCK.

Reply:
Okay, write line about kad country contact you have, not kad pop you have.
Its how it intended to be.

Reply:
It's not intended to be that way anymore.

Reply:
Its intended to be it way it was

Reply:
I'm still mad at this.

Reply:
Also, I changed it so that it mentions humanity in general, not the Kad Republic or Kadmen.

Reply:
I have simple logic here:
You encounter thing A, ju ahets say A is primitive.
You contact with kad - they say kad is primitive.